source/Octopus.TestPortForwarder/PortForwarderBuilder.cs

```diff
@@ -36,19 +37,31 @@ public PortForwarderBuilder WithDataObserver(Func<BiDirectionalDataTransferObser
             return this;
         }
 
+        /// <summary>
+        /// If possible avoid using this as it may not be possible to bind to the given port.
+        /// </summary>
+        /// <param name="listeningPort"></param>
+        /// <returns></returns>
         public PortForwarderBuilder ListenOnPort(int? listeningPort)
         {
             this.listeningPort = listeningPort;
             return this;
         }
 
+        public PortForwarderBuilder WithDelaySendingLastNBytes(int numberOfBytesToDelaySending)
```

Comment:
Nit: Not a big deal. But now that we renamed to `numberOfBytesToDelaySending`, would it be nicer to rename this method to something similar too?

Reply:
yes done.